lib/vfscore/main.c

```diff
@@ -1318,6 +1318,59 @@ UK_SYSCALL_R_DEFINE(int, rename, const char*, oldpath, const char*, newpath)
 	return -error;
 }
 
+UK_TRACEPOINT(trace_vfs_renameat, "\"%s\" \"%s\"", const char*, const char*);
+UK_TRACEPOINT(trace_vfs_renameat_ret, "");
+UK_TRACEPOINT(trace_vfs_renameat_err, "%d", int);
+
+UK_SYSCALL_R_DEFINE(int, renameat, int, olddirfd, const char*, oldpath, int, newdirfd, const char*, newpath) {
+	char src[PATH_MAX];
+	char dest[PATH_MAX];
```

Comment:
A few coding style quibbles:
- please put the opening brace [on its own line](https://unikraft.org/docs/contributing/coding-conventions#braces) (old vfscore code is notoriously inconsitent w/ this, but it's worth keeping new code compliant)
- the function definition spans more than 80 cols, please split